.swiftlint.yml

```diff
@@ -1,11 +1,16 @@
 included:
-    - Source
-    - iOS Example Frame
-    - iOS Example Frame Carthage
-    - iOS Example Frame SPM
+    - "Checkout"
+    - "Source"
+    - "iOS Example Frame"
+    - "iOS Example Frame SPM"
     
 excluded:
-    - iOS Example Frame/Pods
+    - "iOS Example Frame/Pods"
+    - "Checkout/Samples/CocoapodsSample/Pods"
+    - "CheckoutTests"
+    - "Tests"
+    - "iOS Example Frame SPM/iOS Example Frame Regression Tests"
+    - "iOS Example Frame/iOS Example FrameUITests"
```

Comment:
These are added for local checks only. Even if they are not updated, the behaviour of showing the linter errors on GitHub UI would be the same since we only get the diff from the HEAD branch and the main branch and run the lint on those files